packages/mui-joy/src/Autocomplete/AutocompleteProps.ts

```diff
@@ -269,6 +269,10 @@ type AutocompleteOwnProps<
      * @default 'No options'
      */
     noOptionsText?: React.ReactNode;
+    /**
+     * Callback fired when scroll bar reaches bottom of listbox.
+     */
+    onScrollToBottom?: (event: React.SyntheticEvent) => void;
```

Comment:
No need to define it again here since `useAutocomplete` already has it and Joy UI and Material UI extends the hook's types.
```suggestion
```